feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.spec.ts

```diff
@@ -113,15 +176,271 @@ describe('ConfigurationOverviewMenuComponent', () => {
       component.navigateToGroup(GROUP_PREFIX, GROUP_ID_LOCAL);
       expect(
         configuratorStorefrontUtilsService.createOvGroupId
-      ).toHaveBeenCalledWith(GROUP_PREFIX, GROUP_ID_LOCAL);
+      ).toHaveBeenCalled();
     });
 
     it('should invoke utils service for scrolling', () => {
       initialize();
       component.navigateToGroup(GROUP_PREFIX, GROUP_ID_LOCAL);
       expect(
         configuratorStorefrontUtilsService.scrollToConfigurationElement
-      ).toHaveBeenCalledWith('#' + OV_GROUP_ID);
+      ).toHaveBeenCalledWith('#' + OV_GROUP_ID + ' h2');
+    });
+  });
+
+  describe('getGroupId', () => {
+    it('should dispatch request to utils service', () => {
+      initialize();
+      expect(component.getGroupId('A', 'B')).toBe('A--B-ovGroup');
+    });
+
+    it('should cope with undefined idPrefix', () => {
+      initialize();
+      expect(component.getGroupId(undefined, 'B')).toBe('B-ovGroup');
```

Comment:
Strict mode issue. 
Please fix all strict mode issues in this test. Please consider to move to VSCode (there you see them directly), we are facing this issue very often

Reply:
Done!